Src/GoogleApis.Auth.DotNet4/OAuth2/ServiceAccountCredential.cs

```diff
@@ -177,6 +191,16 @@ private static string CreateSerializedHeader()
             return NewtonsoftJsonSerializer.Instance.Serialize(header);
         }
 
+        /// <summary>Converts the PKCS8 PrivateKey to RSA Parameters. Uses the Bouncy Castle library.</summary>
+        private static RSAParameters ConvertPKCS8ToRSAParameters(string pkcs8PrivateKey)

```

Comment:
The tests are covered indirectly as part of the DefaultCredentialProvider tests.